src/obb/server/routes.clj

```diff
@@ -0,0 +1,13 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.routes
+  "Specifices the routes and associates handlers"
+  (:require
+    [compojure.core :as compojure :refer [GET]]
+    [compojure.route :as route]
+    [obb.server.handlers.index :as index]))
+
+(compojure/defroutes public-routes
+  "The routes available to be served, that don't need auth"
+  (GET "/" request (index/handler request))
```

Comment:
Really nitpicking here, but I'd use a verb for this (`handle` instead of `handler`). `handler` gives me the impression that a handler function is being returned, and not that the handler is being called. Disregard if you don't agree.